packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -232,16 +233,17 @@ export function publish(options: InitOptions): void {
             }
           }
         }
-        if (event.isTrusted) {
-          flowletManager.pop(flowlet);
+        if (event.isTrusted && flowletMap.has(eventName)) {
```

Comment:
Do you need the `isTrusted` now again?  I noticed that condition above has also `event.bubbles`. So, we can keep that list of conditions in one place, and only check the map here. 

Also, `.has` and `.get` have similar complexity. So, better to call `.get` once and then check null instead of `.has`